onnxruntime/python/onnxruntime_pybind_exceptions.h

```diff
@@ -43,48 +45,9 @@ struct EPFail : std::runtime_error {
   explicit EPFail(const std::string& what) : std::runtime_error(what) {}
 };
 
-inline void RegisterExceptions(pybind11::module& m) {
-  pybind11::register_exception<Fail>(m, "Fail");
-  pybind11::register_exception<InvalidArgument>(m, "InvalidArgument");
-  pybind11::register_exception<NoSuchFile>(m, "NoSuchFile");
-  pybind11::register_exception<NoModel>(m, "NoModel");
-  pybind11::register_exception<EngineError>(m, "EngineError");
-  pybind11::register_exception<RuntimeException>(m, "RuntimeException");
-  pybind11::register_exception<InvalidProtobuf>(m, "InvalidProtobuf");
-  pybind11::register_exception<ModelLoaded>(m, "ModelLoaded");
-  pybind11::register_exception<NotImplemented>(m, "NotImplemented");
-  pybind11::register_exception<InvalidGraph>(m, "InvalidGraph");
-  pybind11::register_exception<EPFail>(m, "EPFail");
-}
+void RegisterExceptions(pybind11::module& m);
```

Comment:
is there value in we just moving these declarations into onnxruntime_pybind_state_common.h and removing this file altogether? The definitions live in onnxruntime_pybind_state_common.cc anyway ?

Reply:
Discussed offline